### PR TITLE
Defias Brotherhood

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Added the Defias Brotherhood in the 603 bookmark together with Edwin and Vanessa VanCleef, Bazil Thredd, Klaven Mortwake.
- The Syndicate and the Defias Brotherhood AI don't join the Horde/Alliance.

# How to test:
Start in 603, check the Defias Brotherhood, tell what you think.

# Note
I though about adding quick-made government type something like **_Criminal_** for the Defias Brotherhood and the Syndicate with just a few small features like ability to hold cities. Tell me if it's a good idea.

### CODE REVIEW

File: localisation/00_wc_ruler_titles.csv [hunk 1]
```diff
@@ -81,5 +81,7 @@ OVERLORD_TITLE;Overlord;Suzerain;Lehensherr;;;;;;;;;;;x
 OVERLORD_FEMALE_TITLE;Overlord;Suzeraine;Lehensherrin;;;;;;;;;;;x
 HERALD_TITLE;Herald;;;;;;;;;;;;;x
 HERALD_FEM_TITLE;Herald;;;;;;;;;;;;;x
-HIGHFATHER_TITLE;Highfather;;;;;;;;;;;;;
-HIGHFATHER_FEM_TITLE;Highmother;;;;;;;;;;;;;
+HIGHFATHER_TITLE;Highfather;;;;;;;;;;;;;x
+HIGHFATHER_FEM_TITLE;Highmother;;;;;;;;;;;;;x
+KINGPIN_TITLE;Kingpin;;;;;;;;;;;;;x
+KINGPIN_FEM_TITLE;Kingpin;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_wc_ruler_titles.csv]:
```suggestion
KINGPIN_FEM_TITLE;Queenpin;;;;;;;;;;;;;x
```

File: history/characters/1_azerothian.txt [hunk 2]
```diff
@@ -12373,4 +12373,78 @@
...
+
+#dynasty=412
+5815={
+	name=Klaven
+	dynasty=412
+	dna="mortusklavn"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=5 stewardship=5 intrigue=6 learning=7
+	add_trait=scholarly_theologian
+	567.1.15={ birth=yes  add_trait=creature_human }
```
Comment (MemeWizard) [inline history/characters/1_azerothian.txt]: > Klaven Mortwake is a human member of the Defias Brotherhood found in his tower in the Dust Plains, in the human territory of Westfall. He is very unstable mentally, as evidenced by his journal.

Maybe we should give him lunatic trait?

Comment (zumbak04) [reply history/characters/1_azerothian.txt]: Done in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/42ef4e94ffc962b44dd07a92441bf535e53a64a3.

**Comment by zumbak04** [comment]
> Maybe also decision for rogues to adopt "Criminal" government?

Hmmmmmm... like class ability?

**Comment by MemeWizard** [comment]
> Hmmmmmm... like class ability?

Yeah, something like that

**Comment by zumbak04** [comment]
> Yeah, something like that

Hmm, ok, so here's main idea: faster fabricate claims, spend prestige on claims instead of gold, can hold cities. Indepedent rogues can adapt Criminal government to fabricate claims and form big kingdom. Prefer to reform to Monarchy when it's big because can't hold a lot of vassals.

**Comment by sejtalk** [comment]
> > Yeah, something like that
> 
> Hmm, ok, so here's main idea: faster fabricate claims, spend prestige on claims instead of gold, can hold cities. Indepedent rogues can adapt Criminal government to fabricate claims and form big kingdom. Prefer to reform to Monarchy when it's big because can't hold a lot of vassals.

Maybe block them from forming kingdoms so the are good when small but need to reform?

**Comment by zumbak04** [comment]
> Maybe block them from forming kingdoms so the are good when small but need to reform?

Maybe... or insta reform them to Monarchy when they get kingdom title.

**Comment by sejtalk** [comment]
> > Maybe block them from forming kingdoms so the are good when small but need to reform?
> 
> Maybe... or insta reform them to Monarchy when they get kingdom title.

how about no

**Comment by MemeWizard** [comment]
> insta reform them to Monarchy when they get kingdom title

Insta reform will confuse players, I already can see questions in discord like "why I lost my Criminal government type when I created a kingdom".

Also, maybe maximal rank should be limited to kingdoms only, so they cannot create empires.

> Hmm, ok, so here's main idea: faster fabricate claims, spend prestige on claims instead of gold, can hold cities. Indepedent rogues can adapt Criminal government to fabricate claims and form big kingdom. Prefer to reform to Monarchy when it's big because can't hold a lot of vassals.

How about tax penalty and levy bonus? There will be more armed people who can work for you, but also take their "wage" yourself.

**Comment by MemeWizard** [comment]
> spend prestige on claims instead of gold

Didn't get that part
What type of claims are you talking about?
IIRC, they all require both gold and prestige

**Comment by zumbak04** [comment]
> Also, maybe maximal rank should be limited to kingdoms only, so they cannot create empires.

Hmm, not sure. The goals of our historical "Criminal" governments are kingdoms. The Syndicate - Alterac, the Defias - Stormwind.

> How about tax penalty and levy bonus? There will be more armed people who can work for you, but also take their "wage" yourself.

Sounds :ok_hand: 

> > spend prestige on claims instead of gold
> 
> Didn't get that part
> What type of claims are you talking about?
> IIRC, they all require both gold and prestige

Chancellor claims. I mean prestige only.

**Comment by MemeWizard** [comment]
> Tell me if it's a good idea.

Maybe this government also should allow looting regardless of religion and culture.

**Pull request merged**